bigquery_datatransfer/samples/create_scheduled_query.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-bigquery-datatransfer
+
+
+def sample_create_transfer_config(project_id, dataset_id, authorization_code=""):
+    # [START bigquerydatatransfer_create_scheduled_query]
+    from google.cloud import bigquery_datatransfer_v1
+    import google.protobuf.json_format
+
+    client = bigquery_datatransfer_v1.DataTransferServiceClient()
+
+    # TODO(developer): Set the project_id to the project that contains the
+    #                  destination dataset.
+    # project_id = "your-project-id"
+
+    # TODO(developer): Set the destination dataset. The authorized user must
+    #                  have owner permissions on the dataset.
+    # dataset_id = "your_dataset_id"
+
+    # TODO(developer): The first time you run this sample, set the
+    # authorization code to a value from the URL:
+    # https://www.gstatic.com/bigquerydatatransfer/oauthz/auth?client_id=433065040935-hav5fqnc9p9cht3rqneus9115ias2kn1.apps.googleusercontent.com&scope=https://www.googleapis.com/auth/bigquery%20https://www.googleapis.com/auth/drive&redirect_uri=urn:ietf:wg:oauth:2.0:oob
+    #
+    # authorization_code = "_4/ABCD-EFGHIJKLMNOP-QRSTUVWXYZ"
+    #
+    # You can use an empty string for authorization_code in subsequent runs of
+    # this code sample with the same credentials.
+    #
+    # authorization_code = ""
+
+    parent = client.project_path(project_id)
+
+    transfer_config = google.protobuf.json_format.ParseDict(
+        {
+            "destination_dataset_id": dataset_id,
+            "display_name": "Your Scheduled Query Name",
+            "data_source_id": "scheduled_query",
+            "params": {
+                "query": "SELECT 1;  -- Use standard SQL syntax.",
```

Comment:
Let's show the injectable run params that DTS provides.  The version of the sample I wrote did this:

SELECT
  CURRENT_TIMESTAMP() as current_time,
  @run_time as intended_run_time,
  @run_date as intended_run_date,
  17 as some_integer

Reply:
Done in 6b82d54f219d875b0c63a40445e8359053a55580.